standalone/gdre_main.gd

```diff
@@ -97,6 +97,9 @@ func dump_files(exe_file:String, output_dir:String, enc_key:String = "") -> int:
 		# Did 3.3 add new GDScript features?
 		elif version.begins_with("3.3"):
 			decomp = GDScriptDecomp_5565f55.new()
+		# Don't know any changes between 3.3 and 3.4
```

Comment:
There are no byte code changes in `3.3`, `3.4` or `3.5`, so comments (this one, and the previous one) can be removed.